source/app/utils/redaction.js

```diff
@@ -4,9 +4,11 @@ export const normalizeRedactionResponse = (redactions) => {
   const normalizedRedactions = {};
 
   redactions.forEach((redaction) => {
-    if (!normalizedRedactions[redaction.page]) normalizedRedactions[redaction.page] = {};
+    const page = redaction.page.toString()
```

Comment:
Just Curious: Why did we have to break the previous logic into 2 statements & add toString() for each page?

Reply:
I believe the value is used in the subsequent section of the code

Reply:
The front end codebase expects the page number as a string but the back end expects a number